src/Core/Repositories/EntityFramework/UserRepository.cs

```diff
@@ -90,7 +104,7 @@ public async Task UpdateStorageAsync(Guid id)
             {
                 var dbContext = GetDatabaseContext(scope);
                 var ciphers = await dbContext.Ciphers.Where(e => e.UserId == id).ToListAsync();
-                var storage = ciphers.Sum(e => e.AttachmentsJson?.RootElement.EnumerateArray()
+                var storage = ciphers.Sum(e => JsonDocument.Parse(e.Attachments)?.RootElement.EnumerateArray()
```

Comment:
same comment on `null`ness and exception potential.

Reply:
addressed with a null check, though this method has been moved to the base repo